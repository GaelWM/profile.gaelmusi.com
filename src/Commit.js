import React from "react";

const Commit = props => {
  return (
    <li key={props.index} className="list-group-item list-group-item-action">
      {props.message}
    </li>
  );
};

export default Commit;
