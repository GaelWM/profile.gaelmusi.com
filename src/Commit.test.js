import React from "react";
import Commit from "./RepoCommits";
import renderer from "react-test-renderer";

it("renders the commits correctly", () => {
  const props = { index: "1", message: "Modify the login panel" };

  const commit = renderer.create(<Commit props={props} />);
  expect(commit).toMatchSnapshot();
});
