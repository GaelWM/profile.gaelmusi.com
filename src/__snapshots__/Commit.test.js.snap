// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the commits correctly 1`] = `
<div
  className="card-panel"
>
  <ol
    className="list-group"
  />
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Get Last Commits
  </button>
</div>
`;
